lib/ui.js

```diff
@@ -46,11 +49,25 @@ const printCommitLog = async repoUrl => {
 		})
 		.join('\n');
 
+	const changeLog = nextTag => 'Changes: \n' +
+		log.split('\n')
+			.map(commit => {
+				const splitIndex = commit.lastIndexOf(' ');
+				const commitMessage = util.linkifyIssues(repoUrl, commit.slice(0, splitIndex));
+				const commitId = `${repoUrl}/commit/${commit.slice(splitIndex + 1)}`;
```

Comment:
We don't need the full commit URL in the release notes. The Git SHA is enough as GitHub will linkify it for us.

Reply:
I thought using `commit.slice(splitIndex + 1)` is already the Git SHA not the full URL?

Reply:
Yup

Reply:
so the new change is good?

Reply:
Looks good 👌